composer.json

```
@@ -24,6 +24,7 @@
 		"php": ">=7.1",
 		"bjeavons/zxcvbn-php": "^1.1.0",
 		"altis/browser-security": "^1.1.1",
+		"humanmade/disable-accounts": "~0.2.0",
```

```suggestion
		"humanmade/disable-accounts": "^0.2.1",
```

Following feedback and [0.2.1 release](https://github.com/humanmade/disable-accounts/releases/tag/0.2.1)